ui/app/src/components/PreviewConcierge/PreviewConcierge.tsx

```diff
@@ -298,6 +308,7 @@ export function PreviewConcierge(props: PropsWithChildren<{}>) {
   const store = useStore<GlobalState>();
   const { id: siteId, uuid } = useActiveSite() ?? {};
   const user = useActiveUser();
+  const permissions = user?.permissionsBySite[siteId];
```

Comment:
Actually, this is part of the user object which is already in the updated refs. There's no point on having this declared outside of the effect, is there?

Reply:
no point, using user from refs to get permissions